packages/cli/test/integration.test.ts

```diff
@@ -352,24 +355,24 @@ test.after.always(async () => {
   }
 });
 
-test('default command should prompt login with empty auth.json', async t => {
+test('default command should prompt login with empty auth.json', async done => {
   await fs.writeFile(getConfigAuthPath(), JSON.stringify({}));
   try {
     await execa(binaryPath, [...defaultArgs]);
-    t.fail();
+    done.fail();
   } catch (err) {
-    t.true(
+    expect(
       err.stderr.includes(
         'Error: No existing credentials found. Please run `vercel login` or pass "--token"'
       )
-    );
+    ).toBe(true);
```

Comment:
Nit pick: this should use `expect(err.stderr).toContain(msg)`

Reply:
Cleanup PR: https://github.com/vercel/vercel/pull/9667